api.planx.uk/tests/mocks/inviteToPayMocks.ts

```diff
@@ -127,6 +127,11 @@ export const createPaymentRequestQueryMock = {
     payeeName: payee.name,
     payeeEmail: payee.email,
     sessionPreviewData: sessionPreviewData,
+    govPayMetadata: [
+      { key: "source", value: "PlanX" },
+      { key: "isInviteToPay", value: true },
```

Comment:
nit: I'm still find it quite confusing that `isInviteToPay` currently corresponds to whether the PlanX Pay node was _allowing_ invite to pay, rather than if this payment _was paid_ via the invite to pay journey. 

This value came through as `true` on both of my journeys, whereas I'd only expect it to be true for my second when I generated & paid via a payment request. Find the incorrect "regular" pay journey data on this pizza here: 
```
query MyQuery {
  lowcal_sessions_by_pk(id: "d10a5ebb-f1f8-4967-96c2-81eaab3b8182") {
    data (path: "govUkPayment")
  }
}
```